src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -359,22 +359,61 @@
                                                                     Background="Transparent"
                                                                     BorderThickness="0,0,0,0"
                                                                     ContentAlignment="Left"
-                                                                    CornerRadius="0">
-                                                                    <StackPanel Orientation="Vertical">
-                                                                        <custom:CheckBoxWithDescriptionControl
-                                                                            Margin="56,0,0,0"
-                                                                            Description="{x:Bind Path=DisplayDescription}"
-                                                                            Header="{x:Bind Path=DisplayLabel}"
-                                                                            IsChecked="{x:Bind Path=Value, Mode=TwoWay}"
-                                                                            Visibility="{x:Bind Path=ShowCheckBox, Converter={StaticResource BoolToVisibilityConverter}}" />
-                                                                        <ComboBox
+                                                                    CornerRadius="0"
+                                                                    Visibility="{x:Bind Path=ShowCheckBox, Converter={StaticResource BoolToVisibilityConverter}}" >
+                                                                    <custom:CheckBoxWithDescriptionControl
                                                                             Margin="56,0,0,0"
                                                                             Description="{x:Bind Path=DisplayDescription}"
                                                                             Header="{x:Bind Path=DisplayLabel}"
+                                                                            IsChecked="{x:Bind Path=Value, Mode=TwoWay}" />
+                                                                </controls:SettingsCard>
+                                                                <controls:SettingsCard
+                                                                    MinHeight="0"
+                                                                    Margin="56,8,45,8"
```

Comment:
Note to me: For combined version in seperate settings card group update left padding to match with checkbox text.

(Same on TextBox and NumberBox.)